### DISCUSSION TITLE
Problema con scanf

### DISCUSSION THREAD

**Discussion by AnGys09** [INGCOM-UNRN-P1/cursada-2023#66] [Preguntas y respuestas] [answered]
Hola como va? estoy haciendo las correcciones que me pidieron del tp1, pero ahora me encuentro estancada con el ejercicio tres, ya que esta es una parte del codigo : 

```C
int main()
{
    int numeros;
    int numreturn;

    printf("Ingrese un Numero: \n");
    scanf(" %d ", &numeros);
    fflush(stdin);
    numreturn=signo(numeros);

    if(numreturn == -1)
    {
         printf("El numero ingresado es NEGATIVO\n");
```

como se puede ver, tiene un solo SCANF que solicita un numero de tipo entero, hasta ahi genial, el tema es que cuando lo corro al programa me pide que escriba 2 veces el numero y guarda solo el primero, no se como  corregir eso, ya me lei 20 mil paginas pero no se como arreglarlo....  paso lo que me dice la terminal : 

```PS
PS C:\Users\angy_\GIT Programacion 2023\TP01\tp1-2023-AnGys09\output> & .\'ejercicio3.exe'

Ingrese un Numero: 
3
 

9
El nuemro ingresado es POSITIVO

PS C:\Users\angy_\GIT Programacion 2023\TP01\tp1-2023-AnGys09\output> cd 'c:\Users\angy_\GIT Programacion 2023\TP01\tp1-2023-AnGys09\output'
PS C:\Users\angy_\GIT Programacion 2023\TP01\tp1-2023-AnGys09\output> & .\'ejercicio3.exe'
Ingrese un Numero:
0
0
El numero que ingreso es igual a CERO
PS C:\Users\angy_\GIT Programacion 2023\TP01\tp1-2023-AnGys09\output> cd 'c:\Users\angy_\GIT Programacion 2023\TP01\tp1-2023-AnGys09\output'
PS C:\Users\angy_\GIT Programacion 2023\TP01\tp1-2023-AnGys09\output> & .\'ejercicio3.exe'
Ingrese un Numero:
-1
0
El numero ingresado es NEGATIVO
```

**Comment by martinvilu** [comment, marked as answer]
```diff
-scanf(" %d ", &numeros);
+scanf("%d", &numeros);
```
Sin pensarlo mucho, estos espacios me hacen ruido.

(meté el enlace al repositorio así lo veo con el resto del contexto)

**Reply by martinvilu** [reply]
¿Era esto?

Justo ví: https://github.com/INGCOM-UNRN-P1/tp1-2023-AnGys09/commit/2571bd7c4c236ae9a1c4637f66e3de27399b273a

**Reply by AnGys09** [reply]
Gracias lo solucione y me colgue con tildar resuelto!!!!
